type-definitions/most.d.ts

```diff
@@ -0,0 +1,144 @@
+export function observe<A>(f: (a: A) => any, s: Stream<A>): Promise<any>;
+export function forEach<A>(f: (a: A) => any, s: Stream<A>): Promise<any>;
+export function drain<A>(s: Stream<A>): Promise<any>;
+
+export function constant<A, B>(b: B, s: Stream<A>): Stream<B>;
+export function map<A, B>(f: (a: A) => B, s: Stream<A>): Stream<B>;
+export function tap<A>(f: (a: A) => any, s: Stream<A>): Stream<A>;
+export function ap<A, B>(fs: Stream<(a: A) => B>, as: Stream<A> ): Stream<B>;
+export function chain<A, B>(f: (a: A) => Stream<B>, s: Stream<A>): Stream<B>;
+export function flatMap<A, B>(f: (a: A) => Stream<B>, s: Stream<A>): Stream<B>;
+export function join<A>(s: Stream<Stream<A>>): Stream<A>;
+export function switchLatest<A>(s: Stream<Stream<A>>): Stream<A>;
+
+export function continueWith<A>(f: (a: any) => Stream<A>, s: Stream<A>): Stream<A>;
+export function concatMap<A, B>(f: (a: A) => Stream<B>, s: Stream<A>): Stream<B>;
+export function mergeConcurrently<A>(concurrency: number, s: Stream<Stream<A>>): Stream<A>;
+export function merge<A>(...ss: Array<Stream<A>>): Stream<A>;
+export function combine<A>(f: (...args: Array<any>) => A, ...ss: Array<Stream<any>>): Stream<A>;
+
+export function scan<A, B>(f: (b: B, a: A) => B, b: B, s: Stream<A>): Stream<B>;
+export function loop<A, B, S>(f: (seed: S, a: A) => SeedValue<S, B>, seed: S, s: Stream<A>): Stream<B>;
+
+export function cycle<A>(s: Stream<A>): Stream<A>;
+export function concat<A>(s1: Stream<A>, s2: Stream<A>): Stream<A>;
+export function startWith<A>(a: A, s: Stream<A>): Stream<A>;
+
+export function filter<A>(p: (a: A) => boolean, s: Stream<A>): Stream<A>;
+export function skipRepeats<A>(s: Stream<A>): Stream<A>;
+export function skipRepeatsWith<A>(eq: (a1: A, a2: A) => boolean, s: Stream<A>): Stream<A>;
+
+export function take<A>(n: number, s: Stream<A>): Stream<A>;
+export function skip<A>(n: number, s: Stream<A>): Stream<A>;
+export function takeWhile<A>(p: (a:  A) => boolean, s: Stream<A>): Stream<A>;
+export function skipWhile<A>(p: (a: A) => boolean, s: Stream<A>): Stream<A>;
+export function slice<A>(start: number, end: number, s: Stream<A>): Stream<A>;
+
+export function until<A>(signal: Stream<any>, s: Stream<A>): Stream<A>;
+export function takeUntil<A>(signal: Stream<any>, s: Stream<A>): Stream<A>;
+export function since<A>(signal: Stream<any>, s: Stream<A>): Stream<A>;
+export function skipUntil<A>(signal: Stream<any>, s: Stream<A>): Stream<A>;
+export function during<A>(timeWindow: Stream<Stream<any>>, s: Stream<A>): Stream<A>;
+export function throttle<A>(period: number, s: Stream<A>): Stream<A>;
+export function debounce<A>(period: number, s: Stream<A>): Stream<A>;
+
+export function timestamp<A>(s: Stream<A>): Stream<TimeValue<A>>;
+export function delay<A>(dt: number, s: Stream<A>): Stream<A>;
+
+export function fromPromise<A>(p: Promise<A>): Stream<A>;
+export function awaitPromises<A>(s: Stream<Promise<A>>): Stream<A>;
```

Comment:
Heh, oops! Sorry about that :) Is it possible to declare the name `await` in TS? I'll check flow also to see if it complains